src/libraries/System.Security.Cryptography/tests/X509Certificates/X500DistinguishedNameBuilderTests.cs

```diff
@@ -0,0 +1,379 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Formats.Asn1;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    public static class X500DistinguishedNameBuilderTests
+    {
+        private const string TestOid = "2.25.77135202736018529853602245419149860647";
+
+        [Fact]
+        public static void AddEmailAddress_Success_EncodedToIA5()
+        {
+            AssertBuilder(
+                "30223120301E06092A864886F70D01090116116B6576696E406578616D706C652E636F6D",
+                builder => builder.AddEmailAddress("kevin@example.com"));
+        }
+
```

Comment:
Another we-don't-care-about-rfc-822 comment: Probably worth a test that "flibberty gibbet" (or whatever suits you) is a valid email address, as far as the builder is concerned.

Has a space, missing an `@`.  But: totes valid.  Yep, I said "totes".